nodejs/examples/alb/fargateInlineListener/step2/index.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright 2016-2018, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as pulumi from "@pulumi/pulumi";
+import * as aws from "@pulumi/aws";
+import * as awsx from "@pulumi/awsx";
+
+const config = new pulumi.Config("aws");
+const providerOpts = { provider: new aws.Provider("prov", { region: <aws.Region>config.require("envRegion") }) };
+
+const cluster = new awsx.ecs.Cluster("testing", {}, providerOpts);
+
+const service = new awsx.ecs.FargateService("nginx", {
+    cluster,
+    taskDefinitionArgs: {
+        containers: {
+            nginx: {
+                image: "nginx",
+                memory: 128,
+                applicationListener: { port: 80, external: true, loadBalancer: { external: true } }
+            },
+        },
+    },
+    desiredCount: 2,
+}, providerOpts);
+
+export const endpoint = service.listeners.nginx.endpoint;
```

Comment:
Step 2 seems to be the exact copy of Step 1. What is its purpose?

Reply:
got rid of this.  thanks!